apps/src/studio/levels.js

```diff
@@ -3112,4 +3104,65 @@ levels.hoc2015_blockly_14 = utils.extend(levels.js_hoc2015_change_setting,  {
 
 levels.hoc2015_blockly_15 = utils.extend(levels.js_hoc2015_event_free,  {
   editCode: false,
+  startBlocks:
+    '<block type="when_run" deletable="false" x="20" y="20"> \
+      <next> \
+       <block type="studio_setBackground"><title name="VALUE">"endor"</title> \
+        <next> \
+         <block type="studio_setMap"><title name="VALUE">"circle"</title> \
+          <next> \
+           <block type="studio_setSprite"><title name="VALUE">"r2-d2"</title> \
+            <next> \
+             <block type="studio_setDroidSpeed"><title name="VALUE">normal</title> \
+              <next> \
+               <block type="studio_playSound"><title name="SOUND">R2-D2sound5</title></block> \
+              </next> \
+             </block> \
+            </next> \
+           </block> \
+          </next> \
+         </block> \
+        </next> \
+       </block> \
+      </next> \
+     </block>' +
+      whenArrowBlocks(200, 80),
+  toolbox: tb(
+    createCategory(msg.catActions(),
+                    hocMoveNSEW +
+                    blockOfType('studio_setSprite') +
+                    blockOfType('studio_setBackground') +
+                    blockOfType('studio_setDroidSpeed') +
+                    blockOfType('studio_setMap') +
+                    blockOfType('studio_playSound') +
+                    blockOfType('studio_addCharacter') +
+                    blockOfType('studio_setItemSpeed') +
+                    blockOfType('studio_addPoints') +
+                    blockOfType('studio_removePoints') +
+                    blockOfType('studio_endGame')) +
+//    'whenGetStormtrooper': { 'category': 'Events' },
+//    'whenGetRebelPilot': { 'category': 'Events' },
+//    'whenGetPufferPig': { 'category': 'Events' },
+//    'whenGetMynock': { 'category': 'Events' },
+//    'whenGetMouseDroid': { 'category': 'Events' },
+//    'whenGetTauntaun': { 'category': 'Events' },
+//    'whenGetProbot': { 'category': 'Events' },
+//    'whenGetCharacter': { 'category': 'Events' },
+//
+//    'whenGetAllStormtroopers': { 'category': 'Events' },
+//    'whenGetAllRebelPilots': { 'category': 'Events' },
+//    'whenGetAllPufferPigs': { 'category': 'Events' },
+//    'whenGetAllMynocks': { 'category': 'Events' },
+//    'whenGetAllMouseDroids': { 'category': 'Events' },
+//    'whenGetAllTauntauns': { 'category': 'Events' },
+//    'whenGetAllProbots': { 'category': 'Events' },
+//    'whenGetAllCharacters': { 'category': 'Events' }
```

Comment:
These are the list of blocks that are in level 15 droplet that still need to be added to level 15 blockly. I should have been more clear. They will be removed when properly added in the next PR.